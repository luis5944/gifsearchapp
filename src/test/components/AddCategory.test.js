import React from "react";
const { shallow } = require("enzyme");
const { AddCategory } = require("../../components/AddCategory");

describe("Componente AddCategory", () => {
  const setCategories = () => {};
  const wrapper = shallow(<AddCategory setCategories={setCategories} />);
  test("Mostrar el componente de manera correcta ", () => {
    expect(wrapper).toMatchSnapshot();
  });
  test("cambiar la caja de texto", () => {
    const input = wrapper.find("input");
    const value = "Hola Mundo";

    input.simulate("change", { target: { value } });
  });
});
