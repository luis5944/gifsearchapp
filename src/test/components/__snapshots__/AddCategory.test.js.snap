// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente AddCategory Mostrar el componente de manera correcta  1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    onClick={[Function]}
    type="text"
    value="Escribe una categorÃ­a"
  />
</form>
`;
